pkg/helm/export.go

```diff
@@ -26,7 +27,20 @@ func ExportHelmChart(rootFs, fs billy.Filesystem, helmChartPath string, chartVer
 	if err := chart.Validate(); err != nil {
 		return fmt.Errorf("Failed while trying to validate Helm chart: %s", err)
 	}
-	chartVersion = chart.Metadata.Version + chartVersion
+	chartVersionSemver, err := semver.Make(chart.Metadata.Version)
+	if err != nil {
+		return fmt.Errorf("Cannot parse original chart version %s as valid semver", chart.Metadata.Version)
+	}
```

Comment:
Will this fail on the 1.0.000 problem ?

Reply:
If the upstream chart version that was received was itself invalid, then yes I think it would.